fe/fe-core/src/main/java/org/apache/doris/nereids/processor/post/RuntimeFilterGenerator.java

```diff
@@ -497,6 +498,23 @@ public PhysicalRelation visitPhysicalRelation(PhysicalRelation relation, Cascade
         return relation;
     }
 
+    @Override
+    public PhysicalSetOperation visitPhysicalSetOperation(PhysicalSetOperation setOperation, CascadesContext context) {
```

Comment:
why not process it in pushDownRuntimeFilter impl of PhysicalSetOperation?

Reply:
this is not push down, this is to collect alias map